documentation/streams/developer-guide/dsl-topology-naming.html

```diff
@@ -0,0 +1,2 @@
+<!-- should always link the the latest release's documentation -->
```

Comment:
nit: "the" is repeated, but that's not really a big deal (and this is presumably the case for all the other pages too so...)

Reply:
good catch!